### ISSUE TITLE
allow backlinks filter also index specified fields

### ISSUE THREAD

**Issue by tobibeer** [Jermolene/TiddlyWiki5#2020]
extend the [backlinks](http://tiddlywiki.com/#backlinks%20Operator) filter operator to also index (a globally defined) list of user defined fields

**Comment by Jermolene** [comment]
The key thing about backlinks is that it parses the tiddlers to extract the links. I don't think that's quite what you meant though; aren't you after something a bit more like a generalised version of the tagging operator?

**Comment by tobibeer** [comment]
Nope, just links. Right now, I believe, it only indexes links in the text. However, I often use as `summary` field and, it appears, that any links in there get ignored by the backlinks filter.

**Comment by Jermolene** [comment]
Ah, so your proposal is that the `backlinks` filter be enhanced to recognise links within fields other than text? That's quite interesting, yes.

**Comment by tobibeer** [comment]
:+1:
